components/textbox/textbox.jsx

```diff
@@ -42,6 +42,7 @@ export default class Textbox extends React.PureComponent {
         preview: PropTypes.bool,
```

Comment:
Somewhere in this file we would want to check for the group mentions permission in order to not show autocomplete if the user does not have permission to groups

Reply:
wouldn't that be covered by the permission gating of the API call to the store though? In that case the list of groups would be empty and there would be nothing to show.

Reply:
ah true - would and the entire section be hidden if there were no groups right?

Reply:
I will double check, but my understanding is the section header will not be present is there are no items of that type (group mentions here) to suggest.